### PR TITLE
Enable EDA Testing - linters in `tox`

### PR DESCRIPTION
**Pull request by alisonlhart**
Enable approved EDA tests/linters in galaxy-importer to run when running `local_image_build` and in the Openshift container on CRC. The linters will run over the `<collection_root>/extensions/eda/plugins/` dir. 

Linters:
- `ruff`
- `pylint`
- `darglint`

The tox.ini file used in import is not the file PE is distributing to partners. PE is distributing a more CI friendly version of the file, since this one is designed to run in the importer environment. 

Issue: AAH-2307

### CODE REVIEW

**Review by thedoubl3j — APPROVED** [review]
super glad we were able to find a way to get the separate testing logic for pylint. looks great and great work!

**Comment by alisonlhart** [comment]
@awcrosby Need your review/advice. It looks like integration tests are failing due to the tox.ini file not copying correctly into the container in local_image_build.py -> _build_image_with_artifact(). I did some local testing and found that the eda/ dir under galaxy-importer/galaxy-importer/ansible-test/container/eda/ isn't appearing in the Docker container filesystem. My understanding is that anything in the Dockerfile dir or below is built into the container, which seems to be true for the entrypoint.sh file. Even moving the tox.ini file up one dir into the container/ dir doesn't pull it into the Docker build. The logic of how the file is copied in is exactly the same as for the entrypoint.sh file, which is working. 

Any thoughts on this?

**Comment by alisonlhart** [comment]
Resolved the above by adding `ansible_test/container/eda/tox.ini` to `options.package_data` in `setup.py`.

**Comment by alisonlhart** [comment]
PE's thought of how #209, #217 (this), and #218's merge and release could go is:
1. Merge #217 and tag release 0.4.11 for pipeline release before AAP 2.4 date
2. Merge both #209 and #218, and tag release 0.4.12 for pipeline release ~AAP2.4 release date. 

The logic of this would be to get the EDA testing into the pipeline ahead of the 2.4 date, since we will be releasing EDA content onto Automation Hub before the 2.4 release date.

**Review by thedoubl3j — APPROVED** [review]
looks awesome and glad you were able to find the fix for CI/functional tests.

File: galaxy_importer/ansible_test/container/eda/tox.ini [hunk 1]
```diff
@@ -0,0 +1,25 @@
+[tox]
+envlist = ruff, darglint
```
[the remark this answers is not in the document]
Comment (alisonlhart) [reply galaxy_importer/ansible_test/container/eda/tox.ini:2]: I think the envlist can be removed now that all the tests are being called explicitly in entrypoint.sh. I'll double check this and test, then remove it if everything works.

**Review by awcrosby — APPROVED** [review]

**Pull request merged**